hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -35,12 +48,52 @@ Check that the output `0x3020000a` is shown. If it is not, then the OTP bit has
+
+## Raspberry Pi 3B+
+
+The Raspberry Pi 3B+ supports USB Mass Storage boot out of the box. The settings specific to the previous versions of Raspberry Pi do not have to be executed.
+
+This is verified using the following steps, this is the same (procedure)[https://www.raspberrypi.org/documentation/installation/installing-images/] as for SD cards though many alternative ways are functional:
+
+1. Download Raspbian Buster Lite
+
+2. Unzip the .zip file (tested with 7zip, but any other should be fine) to an .img file.
+
+3. Initialise the drive (tested with MBR scheme, no partition/clean disk).
+
+4. Flash the .img file to the external drive (tested using Win32DiskImager on Windows 10)
+
+5. Unmount and disconnect the drive.
+
+6. Connect the drive to the Raspberry Pi and power up the Pi (mind extra USB power usage from external drive).
+
+7. After five to ten seconds, the Raspberry Pi should begin booting and show the rainbow splash screen on an attached display. Make sure that you do not have an SD card inserted in the Pi, since if you do, it will boot from that first.
+
+## Raspberry Pi 4B
+
+Support for USB mass storage boot will be added to the Raspberry Pi 4B in a future software update.
+
+According to rpdom in a [forum post](https://www.raspberrypi.org/forums/viewtopic.php?t=243995#p1488036):
+
+The Pi4 uses a different boot loader to the earlier models. It is stored in eeprom on the board instead of part in the chip and part on the SD card. An update and instructions on how to apply it will be issued when the USB and network boot is ready.
+
+## Extra info
+
+https://www.raspberrypi.org/blog/pi-3-booting-part-i-usb-mass-storage-boot/
 
-Copy a Raspbian or Raspbian Lite installation image directly onto your USB device, in the same way that you would for an SD card. To perform this step, follow the instructions [here](../../../installation/installing-images/README.md), remembering to select the drive that corresponds to your USB mass storage device. Make sure that you use a recent version of Raspbian/Raspbian Lite.
+## Known issues
 
-Once you have finished imaging your USB mass storage device, remove it from your computer and insert it into your Raspberry Pi.
+- The default timeout for checking bootable USB devices is 2 seconds. Some flash drives and rotational harddrives power up too slowly. It’s possible to extend this timeout to five seconds (add a new file 'timeout'to the SD card), but there are devices that fail to respond within this period as well, such as the Verbatim PinStripe 64GB. 
+- Some flash drives have a very specific protocol requirement that we don’t handle; as a result of this, we can’t talk to these drives correctly. An example of such a drive would be the Kingston Data Traveller 100 G3 32G.
+- 3.5" HDD's commonly require 12V as well as 5V and commonly draw too much current for the Pi's USB connections.
 
-## Boot your Raspberry Pi from the USB mass storage device
+## Known working devices
```

Comment:
It'll be impossible to keep lists of "known working devices" or "known not-working devices" up-to-date, so these shouldn't be included in the documentation repo. (same applies to the earlier mentions of "Verbatim PinStripe 64GB" and "")Kingston Data Traveller 100 G3 32G

Reply:
True; It was an attempt at merging the info of the blogpost into this documentation.
Also it would have been interesting info to anyone that has issues or wants to avoid issues, but I agree such list is unmaintainable.

I've removed the list.